website/docs/install/partials/_quickstart-run-beacon-node.md

```diff
@@ -9,118 +9,124 @@ import TabItem from '@theme/TabItem';
         <Tabs groupId="protocol" defaultValue="jwt" values={[
-            {label: 'JWT', value: 'jwt'},
-            {label: 'IPC', value: 'ipc'}
-            ]}>
-                <TabItem value="jwt"><pre><code>./prysm.sh beacon-chain --execution-endpoint=http://localhost:8551 --holesky --suggested-fee-recipient=0x01234567722E6b0000012BFEBf6177F1D2e9758D9 --jwt-secret=jwt.hex --genesis-state=genesis.ssz</code></pre></TabItem>
-                <TabItem value="ipc"><pre><code>./prysm.sh beacon-chain --execution-endpoint=$HOME/.ethereum/&lt;your.ipc&gt; --holesky --suggested-fee-recipient=0x01234567722E6b0000012BFEBf6177F1D2e9758D9 --genesis-state=genesis.ssz</code></pre></TabItem>
-            </Tabs>
+          {label: 'JWT', value: 'jwt'},
+          {label: 'IPC', value: 'ipc'}
+        ]}>
+          <TabItem value="jwt"><pre><code>./prysm.sh beacon-chain --execution-endpoint=http://localhost:8551 --holesky --jwt-secret=&lt;PATH_TO_JWT_FILE&gt; --genesis-state=genesis.ssz</code></pre></TabItem>
+          <TabItem value="ipc"><pre><code>./prysm.sh beacon-chain --execution-endpoint=&lt;PATH_TO_IPC_FILE&gt; --holesky --genesis-state=genesis.ssz</code></pre></TabItem>
+        </Tabs>
       </TabItem>
     </Tabs>
   </TabItem>
 </Tabs>
 
 <div className='hidden-in-jwt-guide hidden-in-mergeprep-guide'>
 
-If you're running a validator, specifying a <code>suggested-fee-recipient</code> wallet address will allow you to earn what were previously miner transaction fee tips. See [How to configure Fee Recipient](../../execution-node/fee-recipient.md) for more information about this feature.
+If you are plannig to run a validator, it is <strong>strongly</strong> advised to use the <code>--suggested-fee-recipient=<WALLET ADDRESS\></code> option. When your validator proposes a block, it will allow you to earn block priority fees, also sometimes called "tips". See [How to configure Fee Recipient](../../execution-node/fee-recipient.md) for more information about this feature.
+
+Your beacon node will now begin syncing from genesis. This usually takes a couple days, but it can take longer depending on your network and hardware specs.
 
-Your beacon node will now begin syncing. This usually takes a couple days, but it can take longer depending on your network and hardware specs.
+It is also possible to sync your beacon node from a checkpoint. This option is considered safer, and will allow your beacon node to sync considerably faster. See [Sync from a checkpoint](../../prysm-usage/checkpoint-sync.md) for more information about this feature.
```

Comment:
since you are working on this anyways could you make checkpoint sync the recommended quickstart instead of from genesis? editing the docs in this way?